tests/dummy/app/routes/nodes/detail/children.js

```diff
@@ -2,10 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
     model() {
-        return this.store.query('node', {
-            filter: {
-                parent: this.modelFor('nodes.detail').get('id')
-            }
-        });
-    }
+        let node = this.modelFor('nodes.detail');
+        return node.get('children');
```

Comment:
I wrote the original method, querying nodes that had a specific parent, which resulted in an APIv2 call to `v2/nodes/?filter[parent]={node_id}`.   The change here made an API v2 request to `v2/nodes/{node_id}/children/`.  It seems like few of the relationships even have this inverse available for querying, so I was just trying to be consistent with other patterns.  Since we have the nodes/node_id/children endpoint, it didn't seem necessary to filter all nodes on parent.

But I did not know the bit about paginating over relationships.